src/python/foglamp/microservice_management/service_registry/instance.py

```diff
@@ -97,8 +97,9 @@ def register(cls, name, s_type, address, port, management_port,  protocol='http'
             else:
                 raise Service.AlreadyExistsWithTheSameName
 
-            if cls.check_address_and_port(address, port):
-                raise Service.AlreadyExistsWithTheSameAddressAndPort
+            # TODO: FOGL-692 - Service Registry should support services that do not have a service port
+            # if cls.check_address_and_port(address, port):
+            #     raise Service.AlreadyExistsWithTheSameAddressAndPort
```

Comment:
This is being dealt with in another Jira task and will cause merge difficulties, please back this change out before checking in. It will be resolved in the merge later.